placetypes/building.json

```diff
@@ -1,9 +1,9 @@
 {
-	"wof:id": 102312327,
+	"wof:id": 1260214523,
```

Comment:
Aside from the fact that WOF ID are immutable once assigned this is a pre-existing WOF ID for a locality in Guinea - https://spelunker.whosonfirst.org/id/1260214523/

Reply:
Ok mais je vois que le nom est incorrecte ce tomba kanssa et non tomba dougoula pouvez vous coorige cela? @tomtaylor

Reply:
You can submit a corrected name using the Write Field editor: https://writefield.nextzen.org/place/edit?url=https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-gn/master/data/126/021/452/3/1260214523.geojson

Reply:
But what a genius thanks to you I manage to propose a modification on the file while waiting for the validation of my modification.  Otherwise I have done everything for months to make changes to his files but I can't thanks to you today I manage to do it I thank you very sincerely for the link you send me and I am  happy I will contact you if necessary thank you very much